usabilla/php-docker-template
# Allow to customize `client_body_buffer_size`

Context
========

Some clients can't handle body split, thus causing nginx to store in the filesystem its cache, which is not desire due to Docker nature, this allows us to customize this directive accordingly to each of the apps.

Need
========

While monitoring feedback collector I've realized many errors related to this

How to test
========

Build and spin up an nginx container, open `/etc/nginx/nginx.conf` and see the configuration in there
